src/content/en/2020/css.md

```diff
@@ -669,15 +715,19 @@ And if you are going for fanciful color names, why not define your own with CSS
   </figcaption>
 </figure>
 
-And, lastly, the once-deprecated, now partially un-deprecated system colors like `Canvas` and `ThreeDDarkShadow`: These were a terrible idea, introduced to emulate the typical user interface of things like Java or Windows 95, and already unable to keep up with Windows 98, they soon fell by the wayside. Some sites use these system colors to try and fingerprint you, a loophole that [we are trying to close as we speak](https://github.com/w3c/csswg-drafts/issues/5710). There are *few good reasons* to use them, and most websites (99.99%) don’t, so we are all good.
+And, lastly, the once-deprecated—now partially un-deprecated—system colors like `Canvas` and `ThreeDDarkShadow`: these were a terrible idea, introduced to emulate the typical user interface of things like Java or Windows 95, and already unable to keep up with Windows 98, they soon fell by the wayside. Some sites use these system colors to try and fingerprint you, a loophole that [we are trying to close as we speak](https://github.com/w3c/csswg-drafts/issues/5710). There are few good reasons to use them, and most websites (99.99%) don't, so we are all good.
 
 The rather useful value `currentColor`, surprisingly, trailed at 0.14% of all sRGB colors (1.62% of all named colors).
 
-All the colors we discussed so far have one thing in common: sRGB, the standard color space for the Web (and for High Definition TV, which is where it came from). Why is that so bad? Because it can only display a limited range of colors: your phone, your TV, and probably your laptop are able to display much more vivid colors due to advances in display technology. Displays with wide color gamut, which used to be reserved for well-paid professional photographers and graphic designers, are now available to everyone. Native apps use this capability, as do digital movies and streaming TV services, but until recently the Web was missing out.
+{# TODO authors—Is it worth explaining why it's useful or is that obvious? #}
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - thoughts on this?

Reply:
I honestly thought it was obvious (but I have the curse of knowledge here). There are lots of colors that we can see, but you can't use them in web pages. Obviously we want to, especially since native apps can use them.

Reply:
I think this is another case where GitHub trying it's best to show the replacement text in red is causing more confusion than help! Ignore the bit in red - I'm asking why `currentColor` is useful and whether that should be explained? If it's the current color why do you need this? Maybe a link to this https://css-tricks.com/currentcolor/ would cover it?

Reply:
Aha! Yes, that link is a good description of why it s useful, compared to just duplicating a color in multiple places.

Reply:
Added that link in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1846/commits/57d05764a596373118a1e666231f9c1e0b0bfca8